test/organizations/CompletionTracking.js

```diff
@@ -0,0 +1,278 @@
+			trackingSpy = sandbox.spy(display, 'updateCompletionTracking');
+			displaySpy = sandbox.spy(display, 'updateDisplayProgress');
+		});
+
+		afterEach(() => {
+			sandbox.restore();
+		});
+
+		it('correct completion parameter', () => {
+			display.updateCompletionTracking(true);
+			expect(trackingSpy.returnValues[0]).to.be.a('promise');
+		});
+		it('correct progress parameter', () => {
+			display.updateDisplayProgress(true);
+			expect(displaySpy.returnValues[0]).to.be.a('promise');
+		});
+		it('incorrect completion parameter', () => {
+			display.updateCompletionTracking(false);
+			expect(trackingSpy.returnValues[0]).to.be.a('promise');
+		});
+		it('incorrect progress parameter', () => {
+			display.updateDisplayProgress(false);
+			expect(displaySpy.returnValues[0]).to.be.a('promise');
+		});
+	});
+
+	describe('update actions, actions missing', () => {
+		let sandbox, trackingSpy, displaySpy, display;
+
+		beforeEach(() => {
+			sandbox = sinon.sandbox.create();
+			display = noActions;
+			trackingSpy = sandbox.spy(display, 'updateCompletionTracking');
+			displaySpy = sandbox.spy(display, 'updateDisplayProgress');
+		});
+
+		afterEach(() => {
+			sandbox.restore();
+		});
+
+		it('true completion parameter', () => {
+			display.updateCompletionTracking(true);
+			expect(trackingSpy.returnValues[0]).to.be.a('promise');
+		});
+		it('true progress parameter', () => {
+			display.updateDisplayProgress(true);
+			expect(displaySpy.returnValues[0]).to.be.a('promise');
+		});
+		it('false parameters', () => {
+			display.updateCompletionTracking(false);
+			expect(trackingSpy.returnValues[0]).to.be.a('promise');
+		});
+		it('false parameters', () => {
+			display.updateDisplayProgress(false);
+			expect(displaySpy.returnValues[0]).to.be.a('promise');
+		});
+	});
+
+	describe('checking update calls', () => {
+		beforeEach(() => {
+			fetchMock.put('http://api.x.io/do/not/track/completion', {});
+			fetchMock.put('http://api.x.io/track/completion', {});
+			fetchMock.put('http://api.x.io/do/not/display/progress', {});
+			fetchMock.put('http://api.x.io/display/progress', {});
+		});
+		afterEach(() => {
+			fetchMock.reset();
+		});
+
+		describe('correct parameters', () => {
+			it('do not track completion', async() => {
+				const display = trackingNoDisplay;
+
+				await display.updateCompletionTracking(false);
+
+				expect(fetchMock.called()).to.be.true;
+				const call = await fetchMock.lastCall();
+				expect(call[0]).to.be.equal('http://api.x.io/do/not/track/completion');
+				const form = await getFormData(call.request);
+				if (!form.notSupported) {
+					expect(form.get('track')).to.equal('false');
+				}
+			});
+
+			it('track completion', async() => {
+				const display = noTrackingDisplay;
+
+				await display.updateCompletionTracking(true);
+
+				expect(fetchMock.called()).to.be.true;
+				const call = await fetchMock.lastCall();
+				expect(call[0]).to.be.equal('http://api.x.io/track/completion');
```

Comment:
`expect(fetchMock.called('http://api.x.io/track/completion')).to.be.true;`  is more readable